### PR TITLE
Swimmer subnet info

### PR DESCRIPTION
**Pull request by quocphu**

### CODE REVIEW

**Comment by ceyonur** [comment]
Could you also update `networks/README.md`?

**Comment by aaronbuchwald** [comment]
> Could you also update `networks/README.md`?

Once this comment is addressed, could you also make sure to sign all of the commits in this PR?

Then I think this PR will be ready to merge.

**Comment by quocphu** [comment]
Hi @ceyonur @aaronbuchwald 
I updated the `network/README.md` file and signed all of my commit.
Could you please help me to review it?

File: networks/README.md [hunk 1]
```diff
@@ -16,9 +16,11 @@ WARNING: this information is provided by the community. Please exercise caution
 | 11111    |    Fuji |                     WAGMI |
 | 432201   |    Fuji |       Dexalot Fuji Subnet |
 | 335      |    Fuji | DeFi Kingdoms Fuji Subnet |
+| 73773    |    Fuji | Swimmer network Fuji Subnet |
```
Comment (ceyonur) [inline networks/README.md:19]:
```suggestion
| 73773    |    Fuji | Swimmer Network Fuji Subnet |
```

File: networks/README.md [hunk 2]
```diff
@@ -16,9 +16,11 @@ WARNING: this information is provided by the community. Please exercise caution
...
 
 ## Mainnet
 
 | Chain ID | Network |                 Name |
 | :------- | ------: | -------------------: |
 | 53935    | Mainnet | DeFi Kingdoms Subnet |
+| 73772    | Mainnet | Swimmer network Subnet |
```
Comment (ceyonur) [inline networks/README.md:26]:
```suggestion
| 73772    | Mainnet | Swimmer Network Subnet |
```

File: networks/mainnet/73772/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Swimmer Mainnet Subnet
+
+## Mainnet Subnet Info
+
+| Swimmer Subnet   |                                                                                            |
+| :-------------------- | ------------------------------------------------------------------------------------------:|
+| Subnet Name           | Swimmer Subnet                                                                             |
+| Subnet Logo           | [subnet logo](https://raw.githubusercontent.com/crabada/logos/dcc185658745ba18511135ff6dae564d57d63768/swimmer/swimmer_icon-512.png)                                        |
```
Comment (ceyonur) [inline networks/mainnet/73772/README.md:8]:
```suggestion
| Subnet Logo           | ![subnet logo](https://raw.githubusercontent.com/crabada/logos/dcc185658745ba18511135ff6dae564d57d63768/swimmer/swimmer_icon-512.png)                                        |
```

File: networks/mainnet/73772/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+| Subnet ID             | 2e3CqLY8thW1WKS89WQ7WR9Qy2bCrrP5mv7psqMRGCHGXuQ1Uj                                          |
+| VM ID                 | srSGD5JeYhL8GLx4RUw53VN5TcoBbax6EeCYmy5S3DiteJhdF                                          |
+| Chain Name            | Swimmer                                                                                    |
+| Chain Logo            | [chain logo](https://raw.githubusercontent.com/crabada/logos/dcc185658745ba18511135ff6dae564d57d63768/swimmer/swimmer_icon-512.png)                                         |
```
Comment (ceyonur) [inline networks/mainnet/73772/README.md:12]:
```suggestion
| Chain Logo            | ![chain logo](https://raw.githubusercontent.com/crabada/logos/dcc185658745ba18511135ff6dae564d57d63768/swimmer/swimmer_icon-512.png)                                         |
```

File: networks/testnet/73773/README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# Swimmer Fuji Subnet
+
+## Fuji Subnet Info
+
+| Swimmer Fuji Subnet   |                                                                                            |
+| :-------------------- | ------------------------------------------------------------------------------------------:|
+| Subnet Name           | Swimmer Subnet                                                                             |
+| Subnet Logo           | [subnet logo](https://raw.githubusercontent.com/crabada/logos/dcc185658745ba18511135ff6dae564d57d63768/swimmer/swimmer_icon-512.png)                                        |
```
Comment (ceyonur) [inline networks/testnet/73773/README.md:8]:
```suggestion
| Subnet Logo           | ![subnet logo](https://raw.githubusercontent.com/crabada/logos/dcc185658745ba18511135ff6dae564d57d63768/swimmer/swimmer_icon-512.png)                                        |
```

File: networks/testnet/73773/README.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+| Subnet ID             | 2WJTQKACZWKAnkakMCafag2nuef6v6UuTFRxH1ndGXKWFg9pPJ                                          |
+| VM ID                 | srSGD5JeYhL8GLx4RUw53VN5TcoBbax6EeCYmy5S3DiteJhdF                                          |
+| Chain Name            | Swimmer                                                                                    |
+| Chain Logo            | [chain logo](https://raw.githubusercontent.com/crabada/logos/dcc185658745ba18511135ff6dae564d57d63768/swimmer/swimmer_icon-512.png)                                         |
```
Comment (ceyonur) [inline networks/testnet/73773/README.md:12]:
```suggestion
| Chain Logo            | ![chain logo](https://raw.githubusercontent.com/crabada/logos/dcc185658745ba18511135ff6dae564d57d63768/swimmer/swimmer_icon-512.png)                                         |
```

File: networks/mainnet/73772/README.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+| Blockchain ID         | 2K33xS9AyP9oCDiHYKVrHe7F54h2La5D8erpTChaAhdzeSu2RX                                          |
+| Eth Chain ID          | 73772                                                                                       |
+| RPC URL               | https://subnets.avax.network/swimmer/testnet/rpc                                           |
```
Comment (ceyonur) [inline networks/mainnet/73772/README.md:15]: I think this is testnet RPC. Could you add your mainnet rpc URL here?

**Review by ceyonur — CHANGES_REQUESTED** [review]

File: networks/mainnet/73772/README.md (hunk 7 above)
Comment (quocphu) [reply networks/mainnet/73772/README.md]: Hi @ceyonur I changed it.

File: networks/README.md (hunk 1 above)
Comment (aaronbuchwald) [inline networks/README.md:19]:
```suggestion
| 73773    |    Fuji | Swimmer Network Fuji Subnet |
```

Comment (aaronbuchwald) [inline networks/README.md:19]:
```suggestion
| 73773    |    Fuji | Swimmer Network Fuji Subnet |
```

**Pull request closed without merging**